gdscript_vn_prototype/dialogue_runner.gd

```diff
@@ -1,4 +1,6 @@
 extends Control
+var storyengine = load("res://lib/StoryEngine/StoryEngine.cs")
+var storynode = storyengine.new("filename", "otherFilename")
```

Comment:
@gailcarmichael woof, sorry for all these whitespace changes and the lack of newline stuff, but these two lines are the changes I actually made (along with copying the lib folder into this project!)

Reply:
This is really cool if it's all it takes to use the C# stuff! I'll have to add some more to the API to test out even if it's just dummy stuff for now.